ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -194,6 +215,19 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-outpost-day2
+- as: aws-ipi-ovn-efs-fips-f28-cert-manager-custom-api-ingress
```

Comment:
leave this to the e2e job owner to decided if need to cover in upgrade ci or not. cc @xingxingxia

Reply:
@jianlinliu thanks, yeah, we'd like to cover it in upgrade. CC @lunarwhite .

Reply:
@JianLi-RH this one is duplicated with the one in openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

Reply:
btw, @xingxingxia this job does not care about arch, right? If right, I think we can move the e2e job to openshift-openshift-tests-private-release-4.17__multi-nightly.yaml to avoid future duplicated PR review questions.

Reply:
@jianlinliu it does not care about arch, it supports arm64 / amd64 / ppc64le / s390x. What do you mean "duplicated PR review questions"? What "duplicated PR review questions" can happen?

Reply:
As far as I know, @JianLi-RH always copy the different ci job from e2e ci jobs to upgrade ci jobs (amd e2e ci -> amd upgrade ci; arm e2e ci -> arm upgrade ci; multi-arch e2e ci -> multi-arch upgrade ci), so we if move the job from amd e2e ci to multi-arch e2e ci, then would avoid duplicated copy.

Reply:
@JianLi-RH OK, I see. `aws-ipi-ovn-efs-fips-f28-cert-manager-custom-api-ingress` currently is only in amd e2e ci, but @JianLi-RH is copying it to amd upgrade ci. But it was already copied to multi-arch upgrade ci by you : ) https://github.com/openshift/release/blob/babfdd1c5a19281a7cf4068fdb611d9c6c1c1d0f/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml#L155 

Therefore @jianlinliu you are suggesting @JianLi-RH not copy it to amd upgrade ci. I agree.
Let me assign @lunarwhite to help move aws-ipi-ovn-efs-fips-f28-cert-manager-custom-api-ingress from the amd e2e ci to multi-arch e2e ci. @lunarwhite could you take it?

Reply:
I can take care of moving the e2e to multi-ci in another PR.

Reply:
Thank you!